src/test/java/org/opentripplanner/street/model/edge/EscalatorEdgeTest.java

```diff
@@ -0,0 +1,44 @@
+package org.opentripplanner.street.model.edge;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.routing.api.request.StreetMode;
+import org.opentripplanner.routing.api.request.preference.WalkPreferences;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.street.model.vertex.SimpleVertex;
+import org.opentripplanner.street.model.vertex.Vertex;
+import org.opentripplanner.street.search.request.StreetSearchRequest;
+import org.opentripplanner.street.search.state.State;
+
+class EscalatorEdgeTest {
```

Comment:
You could add a test where u define some custom value for the escalatorReluctance and test that it affects weight as effected.

Reply:
Here is an example of such a test: https://github.com/opentripplanner/OpenTripPlanner/blob/deac2d5a94db3061c4eb8ca58bc1205450da0e31/src/test/java/org/opentripplanner/street/model/edge/StreetEdgeCostTest.java#L19-L39